package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Service Worker Toolbox provides some simple helpers for use in creating your own service workers.",
   "license": "Apache-2.0",
   "scripts": {
-    "publish-release": "./node_modules/sw-testing-helpers/project/publish-release.sh",
+    "publish-release": "./node_modules/npm-publish-scripts/publish-release.sh",
```

Comment:
It would be cleaner if we merged https://github.com/GoogleChrome/npm-publish-scripts/pull/1 first and then you could just use `"publish-release": "publish-release.sh"` here, instead of having to specify the full path inside of `node_modules/`.